src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css

```
@@ -54,6 +54,14 @@
 * table font sizes
 */
   --pst-table-font-size: 16px;
+
+  /**
+sphinx design font sizes
+*/
```

The formatting of the docstrings should be as follows:

```suggestion
  /**
   sphinx design font sizes
   */
```